test/AspNet.Security.OAuth.Providers.Tests/Keycloak/bundle.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://raw.githubusercontent.com/justeat/httpclient-interception/master/src/HttpClientInterception/Bundles/http-request-bundle-schema.json",
+  "items": [
+    {
+      "uri": "https://{domain}/v1/oauth/token",
```

Comment:
Use a `.local` domain with a fixed value like the Moodle, GitHub Enterprise and Okta tests do.

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dad5906d49eea20886aa35fada099ad8c5043221/test/AspNet.Security.OAuth.Providers.Tests/Okta/bundle.json#L6